### DISCUSSION TITLE
Evil Picture

### DISCUSSION THREAD

**Discussion by DaveH52** [PowerShell/PowerShell#19741] [General]
### Summary of the new feature / enhancement

Did you use an evil-looking character because all administrators are evil?
![PS Character](https://github.com/PowerShell/PowerShell/assets/20617944/d5733497-efe2-459b-8586-294f9e19f010)

**Comment by NoMoreFood** [comment]
I believe the character's name is "Nanoman".  I'm guessing the main goals were to make a character that was "edgy" and gender ambiguous (which I appreciate for inclusivity).  I do agree that the artwork does look a bit "evil" and probably could use a refresh or just be eliminated altogether. After all, PowerShell gets a very bad reputation as being some sort of loner hacking tool to the point where organizations allow any user-downloaded executables and macros to run but PowerShell is locked down hard because it has the perception of being _so_ much more dangerous.

**Comment by kilasuit** [comment]
This isn't really an issue with PowerShell itself so I am converting this to a discussion

**Comment by SydneyhSmith** [comment]
![image](https://github.com/PowerShell/PowerShell/assets/43417619/27e12259-d3ad-497e-a57c-d9685bc80ec1)

**Reply by SydneyhSmith** [reply]
We also have a cuter version 😄
